content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Define Services And Track UI Components In Your Browser Application
+kind: guide
+disable_toc: false
+further_reading:
+- link: "/real_user_monitoring/browser/"
+  tag: "Documentation"
+  text: "RUM Browser Monitoring"
+---
+
+## Overview
+
+RUM lets you capture all events in your browser applications and explore them to troubleshoot slow pages and code errors, or to analyze application usage. All events are available in [RUM Explorer][1] for querying, dashboarding, and alerting.
+
+If your browser application is large, it's likely been built thanks to multiple web development teams. Each team has an area of ownership that they focus on when troubleshooting errors, slowness, or analyzing usage.
+
+This guide describes how to define an application in RUM. In addition, it covers common use cases in large applications where web development teams may require visibility into the health and usage of their area of ownership.
+
+## Create a RUM application
+
+The first step to tracking and analyzing your browser application is to [create a RUM application][2]. A RUM application maps a browser application available at a given domain that renders the experience for what customers would perceive as a website.
+
+For example, if your browser application is available at `https://www.yourwebsite.com`, create a RUM application called "Your Website".
+
+If your browser has subdomains, like `https://www.account.yourwebsite.com`, you should onboard one RUM application for your website.
```

Comment:
I'm confused by this sentence—browsers don't have subdomains. Also, what does it mean to "onboard" a RUM application? Can we just say "create"?

I'm wondering if what was meant by this sentence is "If your browser application has subdomains, like `https://www.account.yourwebsite.com`, create one RUM application for each subdomain."

Reply:
Edited in file. Thanks for catching this!